detox/android/detox/proguard-rules.pro

```diff
@@ -13,12 +13,6 @@
 -dontnote org.hamcrest.**
 -dontnote com.facebook.**
 
-# Detox dynamic access to declared methods of RN's timing module
-
--keep class com.facebook.react.modules.core.TimingModule { *; } # RN >= .63
--keep class com.facebook.react.modules.core.Timing { *; } # RN <= .62
--dontnote com.wix.detox.reactnative.idlingresources.timers.**
```

Comment:
'Was always to wrong place for this, because this only affects the test APK - where RN doesn't exist.